content/docs/04.5-devx/06-apps/05-deploy-app.md

```diff
@@ -0,0 +1,1255 @@
+    registry_uid    = data.spectrocloud_registry.public_registry.id
+    source_app_tier = data.spectrocloud_pack_simple.container_pack.id
+    values          = <<-EOT
+        # ....
+        # ....
+        containerService:
+            serviceName: "{{.spectro.system.appdeployment.tiername}}-svc"
+            registryUrl: ""
+            image: ${var.multiple_container_images["ui"]}
+            access: public
+            ports:
+              - "8080"
+            env:
+              - name: "API_URI"
+                value: "http://{{.spectro.app.$appDeploymentName.api.CONTAINER_SVC_EXTERNALHOSTNAME}}:3000"
+              - name: "TOKEN"
+                value: "${var.token}"
+            serviceType: load balancer
+    EOT   
+  }
+
+```
+
+
+
+<InfoBox>
+
+All container services expose their service address, Kubernetes hostname, and the exposed service ports as output variables.
+You will use output variables frequently when creating app profiles in the future. You can learn more about connecting services by referring to the [Service Connectivity](/devx/app-profile/services/connectivity) documentation.  
+
+</InfoBox>
+
+
+Open the **inputs.tf** file and set the `variable enable-second-scenario"` default value to `true`.
+
+<br />
+
+```terraform
+variable "enable-second-scenario" {
+  type        = bool
+  description = "Whether to enable the second scenario"
+  default     = true
+}
+```
+
+Next, issue the command `terraform apply` to deploy the second scenario. Notice how the `-var` flag is included with the token value in the command. 
+
+<br />
+
+```shell
+terraform apply -var="token=931A3B02-8DCC-543F-A1B2-69423D1A0B94" -auto-approve
+```
+
+```
+// Output condensed for readability
+Apply complete! Resources: 3 added, 0 changed, 0 destroyed.
+```
+
+Log in to [Palette](https://console.spectrocloud.com) and navigate to the left **Main Menu**, click on **Apps**. Select the **scenario-2** row. Once you are on the scenario-2 overview page, click on the exposed URL for the service. A hyperlink for port 8080 and port 3000 is available.
+
+![A view of the scenario-2 overview page](/tutorials/deploy-app/devx_apps_deploy_scenario-2-overview.png)
+
+Click on the UI’s service URL for port **8080** to access the Hello Universe application in a three-tier configuration.
+
+<br />
+
+<WarningBox>
+
+  
+It takes between one to three minutes for DNS to properly resolve the public load balancer URL. We recommend waiting a few moments before clicking on the service URL to prevent the browser from caching an unresolved DNS request.
+
+</WarningBox>
+
+![View of the self-hosted hello universe app](/tutorials/deploy-app/devx_apps_deploy-app_self-hosted-hello-universe.png)
+
+The global counter is no longer available; instead, you have a counter that starts at zero. Each time you click on the center image, the counter is incremented and stored in the Postgres database along with metadata.
+
+
+## Clean-up
```

Comment:
```suggestion
## Cleanup
```

Reply:
@ritawatson We have "clean-up" in the tutorial layout guide.